lib/CL/devices/level0/level0-driver.cc

```diff
@@ -596,20 +599,79 @@ void Level0Queue::writeRect(const void *__restrict__ HostPtr,
   LEVEL0_CHECK_ABORT(res);
 }
 
+
+static void poclLevel0Memfill(Level0Device *Device,
+                                ze_command_list_handle_t CmdListH,
```

Comment:
Is this formatted correctly? Looks like \t tabs?

Reply:
it's not tabs; i changed the name of the function and forgot to re-run clang-format. Fixed